### PR TITLE
Feat/remove extended activity

### PR DESCRIPTION
**Pull request by stiege**

### CODE REVIEW

**Comment by stiege** [comment]
I've squashed a number of commits into one for cleanliness. We do need to get a merge back into master to achieve CI. https://en.wikipedia.org/wiki/Continuous_integration

File: GraceBot.Tests/GraceBot.Tests.csproj [hunk 1]
```diff
@@ -36,22 +36,64 @@
...
+      <HintPath>..\packages\Autofac.3.5.2\lib\net40\Autofac.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
     <Reference Include="Castle.Core, Version=3.3.0.0, Culture=neutral, PublicKeyToken=407dd0808d44fbdc, processorArchitecture=MSIL">
       <HintPath>..\packages\Castle.Core.3.3.3\lib\net45\Castle.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Bot.Connector">
-      <HintPath>..\packages\Microsoft.Bot.Builder.3.0.0\lib\net46\Microsoft.Bot.Connector.dll</HintPath>
+    <Reference Include="Chronic, Version=0.3.2.0, Culture=neutral, PublicKeyToken=3bd1f1ef638b0d3c, processorArchitecture=MSIL">
```
Comment (stiege) [inline GraceBot.Tests/GraceBot.Tests.csproj]: @NightRare @TaoL102 @tommykenji This commit achieves an upgrade of the bot framework from 3 to 3.2

File: GraceBot/App.cs [hunk 2]
```diff
@@ -29,9 +32,7 @@ public App(IFactory factory)
             if (_extendedActivity.Type == ActivityTypes.Message
                 && await _filter.FilterAsync(_extendedActivity))
             {
-                // Save activity to database
-                DbController.SaveActivityToDb(_extendedActivity as ExtendedActivity, ProcessStatus.Unprocessed);
```
Comment (stiege) [inline GraceBot/App.cs]: Dependency injection in order to test the activity is being saved.

File: GraceBot.Tests/GraceBot.Tests.csproj (hunk 1 above)
Comment (NightRare) [inline GraceBot.Tests/GraceBot.Tests.csproj]: Why not moving to the latest version?

**Pull request closed without merging**